src/main/resources/schemas/nhinc/common/NhincCommonAdapter.xsd

```diff
@@ -116,6 +116,24 @@
     </xsd:complexType>
     <xsd:element name="RespondingGateway_CrossGatewayProvideAndRegisterDocumentSetRequestResponse" type="tns:RespondingGateway_CrossGatewayProvideAndRegisterDocumentSetRequestResponseType"/>
     <!-- ====================================================================
+    Document Register Schemas.
+    ==================================================================== -->
+    <xsd:complexType name="RespondingGateway_CrossGatewayRegisterDocumentSetRequestType">
+        <xsd:sequence>
+            <xsd:element ref="ihe:RegisterDocumentSetRequest"/>
+            <xsd:element name="assertion" type="nccommon:AssertionType"/>
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="RespondingGateway_CrossGatewayRegisterDocumentSetRequest" type="tns:RespondingGateway_CrossGatewayRegisterDocumentSetRequestType"/>
+    <xsd:complexType name="RespondingGateway_CrossGatewayRegisterDocumentSetResponseType">
```

Comment:
We do not need this object -->RespondingGateway_CrossGatewayRegisterDocumentSetResponseType